Source/CLI/Help.cpp

```diff
@@ -131,6 +132,14 @@ ReturnValue Help(const char* Name)
     cout << "       --no-check" << endl;
     cout << "              Don't do checks (see above)." << endl;
     cout << endl;
+    cout << "       --info" << endl;
+    cout << "              Provide some extra information about the compresssed file," << endl;
+    cout << "              for example the presence of hash of the raw data." << endl;
+    cout << "              Disable decoding." << endl;
+    cout << "       --no-info" << endl;
+    cout << "              Don't provide some extra information (see above)." << endl;

```

Comment:
The “some” sounds strange to me in a negative sentence. To me `Don't provide extra information` sounds better.

Reply:
word removed.